client/express-checkout/tracking.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { debounce } from 'lodash';
+import { recordUserEvent } from 'tracks';
+
+let expressCheckoutBranding;
+
+// Track the button click event.
+export const trackExpressCheckoutButtonClick = ( source ) => {
+	const expressPaymentTypeEvents = {
+		google_pay: 'gpay_button_click',
+		apple_pay: 'applepay_button_click',
+	};
+
+	const event = expressPaymentTypeEvents[ expressCheckoutBranding ];
+	if ( ! event ) return;
+
+	recordUserEvent( event, { source } );
+};
+
+// Track the button load event.
+export const trackExpressCheckoutButtonLoad = debounce(
+	( { paymentMethods, source } ) => {
+		const expressPaymentTypeEvents = {
+			googlePay: 'gpay_button_load',
+			applePay: 'applepay_button_load',
+		};
+
+		for ( const paymentMethod of paymentMethods ) {
+			const event = expressPaymentTypeEvents[ paymentMethod ];
+			if ( ! event ) continue;
+
+			recordUserEvent( event, { source } );
+		}
+	},
+	1000
+);
+
+export const setExpressCheckoutBranding = ( branding ) =>
+	( expressCheckoutBranding = branding );
```

Comment:
AFAICT this is only called once. Is there a good reason for why we don't just pass `branding` as an optional parameter to the tracking functions?

I never like using globals like this, it feels a little off. It also seems a bit brittle in the sense that we might forget to update it in places and send the wrong tracking event 🤔 

If this is a pattern you followed from a different file I'd strongly suggest not following the previous approach and instead pass `branding` as a parameter to the tracking functions instead of relying on a global.

Reply:
I initially tried to maintain the same function signatures as in the tokenized PRB project to simplify migration. However, since I’ve already modified the signature of the button load event, we should update this one too. I agree that this approach isn’t ideal. 👍

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/8993/commits/b872f1e0cb7b8efa822cfc0b3c7960671c375ab9